lib/models/organisation.js

```diff
@@ -217,98 +217,108 @@ schema.post('init', function handleInit() {
 schema.pre('save', async function preSave(next) {
   const User = getConnection().model('User');
   const Role = getConnection().model('Role');
+  const organisation = this;
+  const objectId = mongoose.Types.ObjectId;
 
   // https://github.com/Automattic/mongoose/issues/1474
   this.wasNew = this.isNew;
 
-  const organisation = this;
-  const objectId = mongoose.Types.ObjectId;
-
   if (!isUndefined(this._oldExpiration) && this.expiration !== this._oldExpiration) {
     this.expirationNotifications.expirationNotificationSent = EMAIL_NOOP;
     this.expirationNotifications.weekBeforeNotificationSent = EMAIL_NOOP;
   }
 
+  if (!isUndefined(this._oldTimezone) && this.timezone !== this._oldTimezone) {
+    await update$dteTimezoneInDB(organisation._id, this.timezone);
+  }
+
+  await User.updateMany(
+    { ownerOrganisation: organisation._id },
+    { ownerOrganisationSettings: this.settings },
+    { runValidators: false },
+  );
+
   // https://github.com/Automattic/mongoose/issues/1474
-  if (organisation.wasNew) {
-    const createAdminRole = await Role.create({
-      title: 'Admin',
-      owner_id: organisation.owner,
-      organisation: organisation._id,
-      scopes: [ALL]
-    });
+  if (!organisation.wasNew) {
+    next();
+    return;
+  }
 
-    const createObserverRole = await Role.create({
-      title: 'Observer',
-      owner_id: organisation.owner,
-      organisation: organisation._id,
-      scopes: [
-        VIEW_PUBLIC_DASHBOARDS,
-        VIEW_PUBLIC_VISUALISATIONS,
-        VIEW_PUBLIC_JOURNEYS
-      ]
-    });
+  const createAdminRole = await Role.create({
```

Comment:
Why are we creating 2 roles? Chris answered me: Just by default. Sorry, I don't understand. 
Is a subset of observer role accesses a part of accesses of administrator role, is not it?

Reply:
We create 2 roles by default one called "Admin" which has access to the Settings and one called "Observer" that does not have access to the Settings. We have done this by default as a convenience for our users. Previously when we didn't have the roles feature we had two types of user "Admin" and "Observer" so the naming is due to legacy to maintain backwards compatibility in our user experience.

Reply:
Okay, I understand. Thank you very much for your answer!